packages/formatting/src/utils/detection.ts

```
@@ -0,0 +1,198 @@
+ */
+function parseVersionOutput(type: FormatterType, output: string): string {
+  const cleanOutput = output.trim();
+
+  switch (type) {
+    case 'prettier':
+      // Prettier outputs just the version number
+      return cleanOutput;
+    case 'biome':
+      // Biome outputs "Version: 1.8.3" or similar
+      const biomeMatch =
+        cleanOutput.match(/Version:\s*(.+)/i) || cleanOutput.match(/(\d+\.\d+\.\d+)/);
```

ESLint

[error] Unexpected lexical declaration in case block.

(no-case-declarations)